crates/rpc/src/method/trace_block_transactions.rs

```diff
@@ -493,6 +493,9 @@ fn map_gateway_function_invocation(
             l1_gas: gas_consumed.l1_gas,
             l2_gas: gas_consumed.l2_gas.unwrap_or_default(),
         },
+        // Pre-0.13.4 failures in individual calls are not possible -- the whole TX is reverted in
+        // that case.
+        is_reverted: false,
```

Comment:
I'm not sure if I get this right but shouldn't there be a TODO here to manage 0.14.0 (ie. the gateway will provide this field)?